tests/integration/test_backup_restore_on_cluster/test_concurrency.py

```diff
@@ -175,11 +174,16 @@ def test_concurrent_backups_on_different_nodes():
 
 @pytest.mark.parametrize(
     "db_engine, table_engine",
-    [("Replicated", "ReplicatedMergeTree"), ("Ordinary", "MergeTree")],
+    [
+        ("Ordinary", "MergeTree"),
+        ("Atomic", "MergeTree"),
+        ("Replicated", "ReplicatedMergeTree"),
```

Comment:
We can also add `("Memory", "MergeTree")` and maybe `("Lazy", "Log")` to make test more interesting